holidayextras/ui-toolkit
# UXUI-222 - New Button styles

#### What does this PR do?
- Adds styles for the ui-componenent-button
- Default, Primary, Secondary, Success, Warning, Danger, Info buttons all added
- Sizes - Small, Large and Extra-Large added (medium is default)
- Updated docs to reflect changes and provided examples of each.

#### What unit or integration tests does this PR have?
N/A

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Check that the code seems sane and that the documentation describes sufficiently how to use it.
`grunt docs` will build the docs for you to see the example etc.

#### How should this be manually tested?
Run `grunt docs` & check the example [http://localhost:4000](http://localhost:4000)
Have a good look at the documentation, you should now see a full range of buttons.
If you click on the `code` tab on the documentation, you can live edit and play with the components.
(N.B this functionality is sketchy, trying clicking on the end of words in the code section to edit)

#### Any background context you want to provide?
Bootstrap buttons are a great base, but we require an extra button. We have used some of bootstraps code but removed elements we deem necessary and added in extra functionality.

#### What are the relevant tickets?
https://holidayextras.jira.com/browse/UXUI-222

#### Screenshots (if appropriate)
![screen shot 2015-04-23 at 11 23 29](https://cloud.githubusercontent.com/assets/778942/7295159/3638ac0a-e9ab-11e4-9827-e91e6629ef88.png)

#### What gif best describes how you feel about this work?
![button moon](http://blog.truffleshuffle.co.uk/wp-content/uploads/2015/02/Button-Moon-Gif.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
